EMS/core-bundle/src/Entity/User.php

```diff
@@ -135,7 +135,7 @@ class User implements UserInterface, EntityInterface, PasswordAuthenticatedUserI
      * @ORM\Column(name="user_options", type="json", nullable=true)
      */
     protected ?array $userOptions = [];
-    private const DEFAULT_LOCALE = 'en';
+    public const DEFAULT_LANGUAGE = 'en';
```

Comment:
To me it could stay DEFAULT_LOCALE

Reply:
https://github.com/ems-project/elasticms/pull/1026/commits/6eadad1f1dd6f98e079947f0ef1e89e292d9f873